R/utils.R

```diff
@@ -209,7 +209,7 @@ phantom_paths <- function() {
   } else {
     path <- '~/bin'
   }
-  path <- c(path, system.file('PhantomJS', package = 'webshot'))
+  path <- c(path, file.path(system.file(package = 'webshot'),'PhantomJS'))
```

Comment:
Tiny style thing: can you please add a space after the comma?

Reply:
```suggestion
  path <- c(path, file.path(system.file(package = 'webshot'), 'PhantomJS'))
```

Reply:
@wch FYI you can directly make suggestions now:

<img width="502" alt="Screen Shot 2019-07-08 at 10 38 13 AM" src="https://user-images.githubusercontent.com/833642/60823172-92074280-a16c-11e9-99e2-1a3dd1352d10.png">

Reply:
I just committed the change.